crates/astria-conductor/src/executor.rs

```diff
@@ -156,12 +161,23 @@ impl<C: ExecutionClient> Executor<C> {
                         );
                         continue;
                     };
-                    if let Err(e) = self.execute_block(block_subset).await {
-                        error!(
-                            height = height,
-                            error = ?e,
-                            "failed to execute block"
-                        );
+                    match self.execute_block(block_subset.clone()).await {
```

Comment:
```suggestion
                    match self.execute_block(block_subset).await {
```

why are we cloning now? looks unnecessary